src/contracts/libraries/BeaconChainProofs.sol

```diff
@@ -376,11 +265,18 @@ library BeaconChainProofs {
     }
 
     /**
-     * @dev Retrieves a validator's withdrawable epoch
+     * @dev Returns true IFF the validator is marked slashed
+     */
+    function getSlashStatus(bytes32[] memory validatorFields) internal pure returns (bool) {
+        return validatorFields[VALIDATOR_SLASHED_INDEX] != 0;
+    }
```

Comment:
might want to just call this `isValidatorSlashed` or something? this kind of naming makes it even more unambiguous that returning 'true' means the validator is slashed

Reply:
fair, the naming was supposed to be in line with the other validatorFields query methods, but you're right that `isValidatorSlashed` is much clearer. Down to change this.

Reply:
[ff7fd32](https://github.com/Layr-Labs/eigenlayer-contracts/pull/515/commits/ff7fd32a609654127efda187f23d3a48bd4b6c1e)